src/services/fiction.ts

```diff
@@ -54,11 +54,6 @@ export interface Review {
     posted: number;
     content: string;
 
-    rating: {
-        up: number;
-        down: number;
-    };
-
```

Comment:
so reviews don't have up/downvotes anymore? if they still do then i would prefer to keep it in the type, otherwise ofc not

Reply:
up/downvotes are hidden now